cmd/context/utils.go

```diff
@@ -400,5 +427,5 @@ func GetManifestV2(basePath, file string) (*model.Manifest, error) {
 	if manifestPath != "" {
 		return model.Get(manifestPath)
 	}
-	return nil, fmt.Errorf("okteto manifest not found")
+	return nil, errors.ErrManifestNotFound
```

Comment:
Overloading the built-in errors package can lead to confusion. This error is fairly specific and probably belongs in a package more focused on manifest. This type of organization would be more like that used with the Go language itself and makes it easier for new code contributors to associate related things while providing isolation from other custom errors we may make that probably belong in similar packages.

Reply:
It's not the built-in errors package, it's own errors package:
https://github.com/okteto/okteto/tree/master/pkg/errors

We are using a single errors package to simplify text errors reviews by non technical people

Reply:
It's not the built-in errors package but we've taken that name which is bad form. It can goes against the grain of what someone familiar with just the built-in package is used to and causes extra cognitive load.

Do we have anyone reviewing the errors? I think I'd rather build the code to have better organization for the heaviest use case, tech reader and implementors. I do think we can solve the non-tech review case in other ways though if it's a real requirement?